### PR TITLE
Add Hardhat to the development guide

### PR DESCRIPTION
**Pull request by mosmartin**
A local Ethereum network designed for development.

### CODE REVIEW

**Review by KukretiShubham — APPROVED** [review]
LGTM !!

File: learning-guide/beginner.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ Click the image to view on YouTube.
 | Title | Content | Length | 
 | --- | :-- | --- |
 | The Complete Guide to Full Stack Ethereum Development - Tutorial for Beginners [2021] | [![video](https://img.youtube.com/vi/a0osIaAOFSE/0.jpg)](https://www.youtube.com/watch?v=a0osIaAOFSE) | 1h 01m 04s
+| Local Solidity Development | https://hardhat.org/ |
```
Comment (KukretiShubham) [inline learning-guide/beginner.md:17]:
```suggestion
| Local Solidity Development | [view](https://hardhat.org/) |
```

**Review by KukretiShubham — COMMENTED** [review]
What you think?

File: learning-guide/beginner.md (hunk 1 above)
Comment (mosmartin) [reply learning-guide/beginner.md:17]: > 

Thanks @KukretiShubham looks good 🙂

**Pull request merged**